docs/DeveloperGuide.md

```diff
@@ -1,12 +1,251 @@
+* The code package command was inspired by a similar package of [@ivanthengwr](https://github.com/ivanthengwr)'s 
+[command package](https://github.com/ivanthengwr/ip/tree/master/src/main/java/duke/command) in his ip. 
+* The style of document in for the developer guide is inspired by the DG of
+[AY2122S1-CS2113T-T10-3](https://github.com/AY2122S1-CS2113T-T10-3/tp/blob/master/docs/DeveloperGuide.md#purpose-of-this-guide)
+
+## Design
+### Architecture
+### Ui component
+### Logic component
+### Model component
+### Storage component
+### Common classes
+
+## Implementation
+
+This section would focus on explaining the application flow and the specifics on how the features are implemented.
+
+List of Commands
+
+The following sequence diagrams to showcase the list of commands, 
+parser() method will not be reflected in order to improve readability.
+
+1. [Add a passenger feature](#Add-a-passenger-feature)
+2. [Delete a passenger feature](#Delete-a-passenger-feature)
+3. [List passengers feature](#List-passengers-feature)
+4. [Add a flight feature](#Add-a-flight-feature)
+5. [Delete a flight feature](#Delete-a-flight-feature)
+6. [List flights feature](#List-flights-feature)
+
+---
+
+### Add a passenger feature
+
+The command Add a passenger adds passenger details of a particular passenger to the passenger list.
+
+**Overview of relevant classes**
+
+**_NOTE:_** Minimal representation of structure of `AddPassengerCommand`
+
+![class diagram](../docs/ug-diagrams/images/passengerAddCmdClassDiagram.jpg)
+
+When the main class `SkyControl` receives a user input, it parses the input with the
+help of the `Parser` class which then forwards the input to the `Passenger Parser` class accordingly.
+The `Passenger Parser` retrieves the command `AddPassengerCommand` and returns it to `SkyControl` where
+further methods will be triggered as explained below.
+
+**Sequence Diagram**
+
+**_NOTE:_** Exceptions are omitted for readability. 
+
+![sequence diagram](../docs/ug-diagrams/images/passengerAddCmdSeqDiagram.jpg)
+
+1. Once the `AddPassengerCommand` is instantiated, the `execute` method is called from the `SkyControl` 
+class with passenger list and user input as method parameters.
+2. Within `AddPassengerCommand` the method `getPassengerDetail(String lineInput)` is called to extract the String 
+consisting of passenger details from the line input.
+3. `AddPassengerCommand` then calls the method `addOperation(String passengerDetail)` within the `PassengerList` class.
+4. `getPassengerDetails(String passengerDetail)` method then extracts each of the passenger detail into an attribute in 
+the `PassengerList` class
+5. `passenger` object of the class `PassengerInfo` is instantiated using the attributes retrieved from the method in 
+Step 4. The `passenger` object is then added to the list of passengers
+6. `Ui` class level method `showAddedPassenger(PassengerInfo passenger)` is used to display to the user that passenger
```

Comment:
Some of the info in these steps are repetitive; they can be read from the diagram itself. Perhaps you could include only the more insightful information